source/intro.tex

```diff
@@ -121,9 +121,7 @@
 \indexdefn{access}%
 \definition{access}{defns.access}
 \defncontext{execution-time action}
-read\iref{conv.lval} or
-modify (\ref{expr.ass}, \ref{expr.post.incr}, \ref{expr.pre.incr})
-the value of an object
```

Comment:
Can we add some text to the note below to retain these useful xrefs? Perhaps add between the first and second sentence:

> Reads of scalar objects are described in \ref{conv.lval} and modifications of scalar objects are describred in \ref{expr.ass}, \ref{expr.post.incr}, and \ref{expr.pre.incr}.

I think it's important to the use of this definition that we know what these four cases are.

Reply:
Fixed.